x-pack/plugins/snapshot_restore/public/application/sections/home/policy_list/policy_list.tsx

```diff
@@ -40,14 +42,14 @@ interface MatchParams {
 export const PolicyList: React.FunctionComponent<RouteComponentProps<MatchParams>> = ({
   history,
 }) => {
+  const queryClient = useQueryClient();
```

Comment:
Could there be a way to obfuscate the query client that is being used from each consumer so if we decided to change the query client under the hood it would be less impactful?